extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -424,10 +426,14 @@ public function saveUserData() {
 			 */
 			$userIdentityBox = new UserIdentityBox( $user );
 
-			if ( !empty( $userData->website ) && 0 !== strpos( $userData->website, 'http' ) ) {
+			if ( !empty( $userData->website ) && strpos( $userData->website, 'http' ) !== 0 ) {
 				$userData->website = 'http://' . $userData->website;
 			}
 
+			if ( !empty( $userData->fbPage ) && strpos( $userData->website, self::FBPAGE_PREFIX ) !== 0 ) {
```

Comment:
copy-paste on second predicate? /s/website/fbPage
Also what if someone adds `facebook.com/...` or `www.facebook.com/...` or `www.fb.com/...`? Maybe it should be specified in hint text besides the input field.